runtime/rasdump/dmpsup.c

```diff
@@ -577,7 +578,32 @@ configureDumpAgents(J9JavaVM *vm)
 		}
 		if ( IS_CONSUMABLE(j9vm_args, xdumpIndex) && !IS_CONSUMED(j9vm_args, xdumpIndex) )
 		{
-			GET_OPTION_VALUE(xdumpIndex, ':', &optionString);
+			/* Handle mapped tool dump options */
+			if (HAS_MAPPING(j9vm_args, xdumpIndex)) {
+				char *toolCursor = strstr(MAPPING_J9NAME(j9vm_args, xdumpIndex), ":tool") + 1;
```

Comment:
For this line, `+ 1` moves the pointer past the colon before `"tool"`, so that it points to `"tool..."` instead of `":tool..."`. This is needed due to how group dump settings are parsed from `optionString`.

However, it prevents the null check from being false. I believe that moving the `+ 1` into the `if` clause should fix this.